code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,66 @@
+package code401.challenges.linkedlist;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class LinkedList {
+    Node head;
+    int size;
+
+    LinkedList() {
+        //upon instantiation the head will point to null
+        this.head = null;
+        size = 0;
```

Comment:
When assigning instance variables Java doesn't require you to use `this`. If you use `this` then you should use it everywhere. Consistency is something to strive for!